cmd/armadactl/cmd/watch_test.go

```diff
@@ -0,0 +1,62 @@
+package cmd
+
+import (
+	"io"
+	"testing"
+
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/require"
+
+	"github.com/armadaproject/armada/internal/armadactl"
+)
+
+func TestWatch(t *testing.T) {
+	// TODO there are no tests for invalid input because cobra silently discards those inputs without raising errors
```

Comment:
Is this a TODO for this PR?

Reply:
Sorry for the comment. This was a mistake from my side. The TODO comment was for another file. This PR doesn't include any TODO comments.